test/support/test_macros.h

```diff
@@ -0,0 +1,31 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SUPPORT_TEST_MACROS_HPP
+#define SUPPORT_TEST_MACROS_HPP
+
+/* Make a nice name for the standard version */
+#ifndef TEST_STD_VER
```

Comment:
I suggest adding "_ONEDPL_TEST_" prefix.

Reply:
Fixed (removed): this define not required anymore